### ISSUE TITLE
Which star is which?

### ISSUE THREAD

**Issue by Trickiy** [TheXTech/TheXTech#803]
### Describe the issue

I can't tell the difference.
![image](https://github.com/user-attachments/assets/f3469649-29b1-4397-bf8b-2c32a450718b)

### A type of bug

Native bug

### Does this issue happens always or randomly?

Always happens

### Version

1.3.7-dev

### Version Hash

#1c43287

### Platform

3DS

### Processor architecture

ARM32 / ARMv7 (32-bit ARM)

**Comment by Superbloxen** [comment]
I wouldn't call this a bug, this just needs indication to tell the SMB3 Star and SMB3 Goal Cards apart. I feel the issue of objects looking identical also extends to all custom graphics and assets that coincide appearance-wise.


Additionally, you shouldn't phrase issues like this in the future, it's better for you to write a concise explanation about said issue rather than writing in a vague and somewhat rude way.

**Comment by ds-sloth** [comment]
That's true, but the indication is normally that the goal card is animated. I'll rename this issue to cover all animated NPCs.

**Comment by Trickiy** [comment]
>Additionally, you shouldn't phrase issues like this in the future, it's better for you to write a concise explanation about said issue rather than writing in a vague and somewhat rude way.

I am sorry I was not aware that saying thing in a vague way is rude, and I said it in a vague way because I new the issue is obvious. But I will moving forward not be vague and and instead drag on the explanation from now on.
